core/symbol/capitalize_spec.rb

```diff
@@ -21,6 +21,13 @@
     end
   end
 
+  ruby_version_is '2.4' do
+    it "capitalizes the first character regardless of if it is Unicode" do
```

Comment:
`regardless` sounds a bit odd to me, but I'm not a native English speaker.
I think it might read better as `capitalizes the first character if it is Unicode`.